muselsl/muse.py

```diff
@@ -40,72 +42,52 @@ def __init__(self, address=None, callback_eeg=None, callback_control=None,
         self.interface = interface
         self.time_func = time_func
 
-        if backend in ['auto', 'gatt', 'bgapi']:
-            if backend == 'auto':
-                if platform == "linux" or platform == "linux2":
-                    self.backend = 'gatt'
-                else:
-                    self.backend = 'bgapi'
-            else:
-                self.backend = backend
-        else:
-            raise(ValueError('Backend must be auto, gatt or bgapi'))
+        self.backend = helper.resolve_backend(backend)
 
     def connect(self, interface=None, backend='auto'):
         """Connect to the device"""
+        try:
+            if self.backend == 'bluemuse':
+                print('Starting BlueMuse.')
+                subprocess.call('start bluemuse:', shell=True)
 
-        if self.backend == 'gatt':
-            self.interface = self.interface or 'hci0'
-            self.adapter = pygatt.GATTToolBackend(self.interface)
-        else:
-            self.adapter = pygatt.BGAPIBackend(serial_port=self.interface)
-
-        self.adapter.start()
-
-        if self.address is None:
-            address = self.find_muse_address(self.name)
-            if address is None:
-                raise(ValueError("Can't find Muse Device"))
             else:
-                self.address = address
-        self.device = self.adapter.connect(self.address)
+                if self.backend == 'gatt':
+                    self.interface = self.interface or 'hci0'
+                    self.adapter = pygatt.GATTToolBackend(self.interface)
+                else:
+                    self.adapter = pygatt.BGAPIBackend(
+                        serial_port=self.interface)
+
+                self.adapter.start()
+                self.device = self.adapter.connect(self.address)
 
-        # subscribes to EEG stream
-        if self.enable_eeg:
-            self._subscribe_eeg()
+                # subscribes to EEG stream
+                if self.enable_eeg:
```

Comment:
does this only apply to gatt or BGAPIBackend? it's currently fitting that scope

Reply:
You can't subscribe like this with bluemuse backend so yea.